.github/actions/paths-filter/filters.yml

```
@@ -1,15 +1,18 @@
 rust:
+  - .github/**
   - oxidation/**/*.rs
   - oxidation/**/*.toml
   - Cargo.toml
   - Cargo.lock
 python:
+  - .github/**
   - parsec/**
   - tests/**
   - pyproject.toml
   - poetry.lock
   - build.py
 rust-ext:
+  - .github/**
   - src/**/*.rs
   - Cargo.toml
   - Cargo.lock
```

```suggestion
rust:
  - oxidation/**/*.rs
  - oxidation/**/*.toml
  - Cargo.toml
  - Cargo.lock
python:
  - parsec/**
  - tests/**
  - pyproject.toml
  - poetry.lock
  - build.py
rust-ext:
  - src/**/*.rs
  - Cargo.toml
  - Cargo.lock
ci:
  - .github/actions/**
  - .github/workflows/**
```
Add another field instead

---

I'd rather stay KISS here (on top of that, listing subfolders on `.github/` means we will have issues if we add something new in the `.github/`directly)

---

maybe, listening to the subfolder on `.github` is to much.
But I still think adding the field `ci` is better

---

>  But I still think adding the field ci is better

I don't see a direct benefit (except from the "we named it "rust" and it contains ci stuff which is not technically Rust code", hence KISS)